src/App.jsx

```diff
@@ -23,7 +23,7 @@ const semiPersistentState = () => {
 
 function App() {
 
-  const [todoList, setTodoList] = semiPersistentState();
+  const [todoList, setTodoList] = usesemiPersistentState();
```

Comment:
we use camel casing so it is easier to read and understand, I will approve the MR but you need to make the changes before you merge

```suggestion
  const [todoList, setTodoList] = useSemiPersistentState();
 ````